3.10/installation-mac-osx.md

```diff
@@ -1,15 +1,16 @@
 ---
 layout: default
-description: ArangoDB under macOS can be installed via Homebrew, DMG Package or tar.gz Archive
+description: You can install ArangoDB on macOS via Homebrew, DMG Package or tar.gz Archive
+title: Installing ArangoDB on macOS
 ---
 Installing ArangoDB on macOS
 ============================
 
-ArangoDB under macOS can be installed via:
+You can install ArangoDB on macOS in different ways:
 
-1. [Homebrew](#homebrew)
-2. [_DMG_ Package](#package-installation)
-3. [_tar.gz_ Archive](#installing-using-the-archive) (starting from v3.4.0)
+- [Homebrew](#homebrew)
+- [_DMG_ Package](#package-installation)
+- [_tar.gz_ Archive](#installing-using-the-archive)
```

Comment:
homebrew has the checksum delivered as part of the pull request when publishing the formular; we should also note this.